pkg/yttlibrary/assert.go

```diff
@@ -36,3 +38,21 @@ func (b assertModule) Fail(thread *starlark.Thread, f *starlark.Builtin, args st
 
 	return starlark.None, fmt.Errorf("fail: %s", val)
 }
+
+func (b assertModule) TryTo(thread *starlark.Thread, f *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if args.Len() != 1 {
+		return starlark.None, fmt.Errorf("expected exactly one argument")
+	}
+
+	lambda := args.Index(0)
+	if _, ok := lambda.(starlark.Callable); !ok {
+		return starlark.None, errors.New("expected argument to be a callable function or lambda")
+	}
+
+	var errVal starlark.Value = starlark.None
+	retVal, err := starlark.Call(thread, lambda, nil, nil)
+	if err != nil {
+		errVal = starlark.String(err.Error())
+	}
+	return starlark.Tuple{retVal, errVal}, nil
```

Comment:
might be worth throwing a test for a function that returns a tuple. (i guess we will have a tuple within a tuple.)